tests/unit/nlp/language_modeling/test_lm_datasets.py

```diff
@@ -1,13 +1,23 @@
-from flambe.nlp.language_modeling import PTBDataset
+# TODO: uncomment after fixing circleCI
```

Comment:
Lets fix this before merging imo. I can help investigate.

Reply:
I don't know how to. You can try